### PR TITLE
Translate adopters

### PR DESCRIPTION
**Pull request by RainbowMango**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
This PR translate [Karmada Adopters Page](https://karmada.io/adopters/), mostly migrate from [v1.6 Adopter list](https://karmada.io/zh/docs/casestudies/adopters/).

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by RainbowMango** [comment]
@SAMZONG
Can you help to take a look at why the build is failing? (I still can't build this site on my side :()

**Comment by samzong** [comment]
> @samzong Can you help to take a look at why the build is failing? (I still can't build this site on my side :()

Okay, I will check at this weekend.

File: i18n/zh/docusaurus-plugin-content-pages/adopters.md [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
\ No newline at end of file
+| 组织/公司名称    | 网站                            | 使用场景                                                                                           | 案例研究                                                                   |
+|------------|-------------------------------|------------------------------------------------------------------------------------------------|------------------------------------------------------------------------|
+| 兴业数金       | https://www.cibfintech.com/   | 使用 Karmada 构建更灵活、更有弹性的混合多云平台                                                                   | 待定                                                                     |
+| 中国电子云      | https://cecloud.com/          | 使用 Karmada 构建多集群平台                                                                             | 待定                                                                     |
+| 道客网络科技     | https://www.daocloud.io/      | 使用 Karmada 构建更灵活、更有弹性的混合多云平台                                                                   | [DaoCloud结合Karmada打造新一代企业级多云平台](daocloud.md)                           |
```
Comment (samzong) [inline i18n/zh/docusaurus-plugin-content-pages/adopters.md:22]: I guess this is a document reference problems, relative and ` yarn build ` executive position, the current directory does not exist daocloud, md, before you can refer to the configuration of the adjustment of relative path for the/docs/casestudies/daocloud md

Comment (RainbowMango) [reply i18n/zh/docusaurus-plugin-content-pages/adopters.md]: Great thanks. It works after updating the link with the link `https://karmada.io/zh/docs/casestudies/daocloud`.

I'll try again with the relative path.

Comment (RainbowMango) [reply i18n/zh/docusaurus-plugin-content-pages/adopters.md]: Seems `/docs/casestudies/daocloud.md` does not work. 
Still `Deploy failed` without any useful log.

**Review by RainbowMango — COMMENTED** [review]
/assign @samzong 

Please help take a look.

Please feel free to make any changes you want.

**Pull request merged**